packages/oauth/oauth_tests.js

```diff
@@ -59,3 +59,51 @@ Tinytest.add(
     test.equal(OAuth._retrievePendingCredential(key, secret), cred);
   }
 );
+
```

Comment:
It would be nice to also add a test for the default (current) `ROOT_URL_PATH_PREFIX` behavior (i.e. blank `''` and not modified).